### PR TITLE
[EchoTask] Fixed type handling.

### PR DESCRIPTION
**Pull request by siad007**
Fixed bool assumption for not typed hinted setters.
Related to #1201

### CODE REVIEW

File: classes/phing/tasks/system/EchoTask.php [hunk 1]
```diff
@@ -116,27 +116,27 @@ protected function getFilesetsMsg()
      *
      * @param $file
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

File: classes/phing/tasks/system/EchoTask.php [hunk 2]
```diff
@@ -116,27 +116,27 @@ protected function getFilesetsMsg()
...
+    public function setFile(string $file)
     {
-        $this->file = (string) $file;
+        $this->file = $file;
     }
 
     /**
      * setter for level
      *
      * @param $level
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

File: classes/phing/tasks/system/EchoTask.php [hunk 3]
```diff
@@ -116,27 +116,27 @@ protected function getFilesetsMsg()
...
+    public function setLevel(string $level)
     {
-        $this->level = (string) $level;
+        $this->level = $level;
     }
 
     /**
      * setter for append
      *
      * @param $append
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

File: classes/phing/tasks/system/EchoTask.php [hunk 4]
```diff
@@ -146,7 +146,7 @@ public function setAppend($append)
      *
      * @param $msg
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

File: classes/phing/tasks/system/EchoTask.php [hunk 5]
```diff
@@ -156,18 +156,18 @@ public function setMsg($msg)
...
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

File: classes/phing/tasks/system/EchoTask.php [hunk 6]
```diff
@@ -156,18 +156,18 @@ public function setMsg($msg)
...
+    public function setMessage(string $msg)
     {
-        $this->msg = (string) $msg;
+        $this->msg = $msg;
     }
 
     /**
      * Supporting the <echo>Message</echo> syntax.
      *
      * @param $msg
```
Comment (mimmi20) [inline classes/phing/tasks/system/EchoTask.php]: Please add the type here.

**Comment by siad007** [comment]
@mimmi20 thx for your review, but your requests are actual not the scope of this bug fix. Please open a new ticket or make a contribution.

**Pull request merged**